### PR TITLE
181 fix records order bptree

### LINKED ISSUE #181: Order of records in bptree after deserialization is reversed
https://github.com/qoollo/pearl/blob/master/src/blob/index/core.rs#L311

Right now after deserialization indexes have records reversed which leads to returning oldest record instead of newest

### PR DESCRIPTION
**Pull request by idruzhitskiy**
Closes #181

### CODE REVIEW

File: src/blob/index/bptree/core.rs [hunk 1]
```diff
@@ -131,6 +131,12 @@ where
                 }
                 Ok(headers)
             })
+            .map(|mut headers| {
+                for val in headers.values_mut() {
+                    val.reverse();
```
Comment (ikopylov) [inline src/blob/index/bptree/core.rs:136]: For the most common case when `val.len() == 1` we can avoid calling `reverse`. Please, add condition before the call

Comment (idruzhitskiy) [reply src/blob/index/bptree/core.rs:136]: I'm surprised this check is not part of the reverse itself. Ok, I'll add this

Comment (ikopylov) [reply src/blob/index/bptree/core.rs:136]: According to the source code (https://doc.rust-lang.org/src/core/slice/mod.rs.html#678) it is not a part

**Review by ikopylov — APPROVED** [review]
LGTM

**Pull request merged**